strawberry/exceptions.py

```diff
@@ -14,6 +14,13 @@ def __init__(self):
         super().__init__(message)
 
 
+class NotAClass(Exception):
```

Comment:
I think a better name for this exception might be something along the lines of `ObjectIsNotTypeError` (especially with the `Error` at the end)